### PR TITLE
Finish Global Funnel damage support

### PR DESCRIPTION
**Pull request by syrifgit**
Support finished and tested for Assassination and Enhancement

### CODE REVIEW

File: TheWarWithin/Priorities/ShamanEnhancement.simc [hunk 1]
```diff
@@ -33,8 +33,8 @@ actions+=/feral_spirit,if=talent.elemental_spirits.enabled|(talent.alpha_wolf.en
 actions+=/surging_totem
 actions+=/ascendance,if=dot.flame_shock.ticking&((ti_lightning_bolt&active_enemies=1&raid_event.adds.in>=action.ascendance.cooldown%2)|(ti_chain_lightning&active_enemies>1))
 actions+=/call_action_list,name=single,strict=1,if=active_enemies=1
-actions+=/call_action_list,name=aoe,strict=1,if=active_enemies>1&(rotation.standard|rotation.simple)
-actions+=/call_action_list,name=funnel,strict=1,if=active_enemies>1&rotation.funnel
+actions+=/call_action_list,name=aoe,strict=1,if=active_enemies>1&(! toggle.funnel)
```
Comment (Hekili) [inline TheWarWithin/Priorities/ShamanEnhancement.simc]: Remove space.

File: TheWarWithin/ShamanEnhancement.lua [hunk 2]
```diff
@@ -2611,13 +2611,14 @@ spec:RegisterOptions( {
...
+    "If funnel is enabled, the default priority will recommend spending resources on single target spenders in AoE to deal high priority damage to an important mob.\n\n",
+    desc = "",
+    type = "description",
+    fontSize = "medium",
     width = "full"
-} )
+})
 
 spec:RegisterStateExpr( "funnel", function()
     return settings.funnel_priority
```
Comment (Hekili) [inline TheWarWithin/ShamanEnhancement.lua]: Update to `toggle.funnel` for legacy support.

File: TheWarWithin/Priorities/ShamanEnhancement.simc (hunk 1 above)
Comment (syrifgit) [reply TheWarWithin/Priorities/ShamanEnhancement.simc]: oops the space is only for the in-game UI

**Comment by syrifgit** [comment]
Reviews incorporated, corrected Guardian druid wrong pack string from @davidmann4's PR

Although I see in a search that the toggle is used for Arcane and Affliction now, I will check those too.

**Pull request merged**